cmd/apps/certmanager_app.go

```diff
@@ -129,7 +173,94 @@ func MakeInstallCertManager() *cobra.Command {
 const CertManagerInfoMsg = `# Get started with cert-manager here:
 # https://docs.cert-manager.io/en/latest/tutorials/acme/http-validation.html`
 
+const AcmeInfoMsg = `# A default ClusterIssuer was installed and configured for acme usage (with kubernetes.io/tls-acme: "true"):
+# https://cert-manager.io/docs/usage/ingress/#optional-configuration`
+
 const certManagerInstallMsg = `=======================================================================
 = cert-manager  has been installed.                                   =
 =======================================================================` +
 	"\n\n" + CertManagerInfoMsg + "\n\n" + pkg.ThanksForUsing
+
+func WriteDefaultIssuer(email string, ingressClass string, staging bool) (err error) {
+	log.Printf("Installing default letsencrypt issuer\n")
+
+	yamlBytes, templateErr := buildDefaultIssuerYAML(email, ingressClass, staging)
+	if templateErr != nil {
+		log.Print("Unable to install the application. Could not build the templated yaml file for the resources")
+		return templateErr
+	}
+
+	tempFile, tempFileErr := writeTempFile(yamlBytes, "temp_registry_ingress.yaml")
+	if tempFileErr != nil {
+		log.Print("Unable to save generated yaml file into the temporary directory")
+		return tempFileErr
+	}
```

Comment:
Imo it would be more reasonable to `fmt.Errorf` the log line, like in line 206 or every other application.

Reply:
Done